src/js/createLoading.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * loadingDiv is a global variable, because I wasn't able
+ * to check if typeof loadingDiv === "undefined".
+ * Normally typeof don't need variable to be initialized,
+ * but for some reason I got an error that there is
+ * no such variable as loadingDiv.
+ * Firefox version: 106.0.2 (64-bit) on Ubuntu 20.04.1
+ */
+loadingDiv = document.createElement("div");
```

Comment:
Is there any reason why this can't be a regular `const` variable?

If that cannot be declared in a global scope, consider using [an IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE)